homeassistant/config.py

```diff
@@ -487,6 +488,10 @@ def _format_config_error(
         f" (See {getattr(domain_config, '__config_file__', '?')}, "
         f"line {getattr(domain_config, '__line__', '?')}). "
     )
+    try:
+        message += f"Entry id: {config.get(CONF_ID)}. Alias: {config.get(CONF_ALIAS)}. "
+    except AttributeError:
```

Comment:
Not clear what can raise `AttributeError` here.

Reply:
Thanks for the review.  This might be an artifact after some test I did before.  
Probably related to the comment regarding the previous lines.

Reply:
I just looked at the code, and it is a mirror of the code a few lines above:
https://github.com/home-assistant/core/blob/4c097d0d36b62235885631b0ca5068390e982cc9/homeassistant/config.py#L482-L485
Because `config` is sometimes a string, and sometimes a dict.

So without this, you sometimes get errors like
```
  File "/workspaces/home-assistant/homeassistant/config.py", line 495, in _format_config_error
    message += f"Entry id: {config.get(CONF_ID)}. Alias: {config.get(CONF_ALIAS)}. "
AttributeError: 'str' object has no attribute 'get'
```